pkg/utils/resource.go

```diff
@@ -0,0 +1,51 @@
+package utils
```

Comment:
`pkg` is a directory that putting all common packages unrelated to the core logic of pouch.

So, maybe we should not put `resource.go` to `pkg`, WDYT?

Reply:
> just move GetThrottleDevice from mgr to ctrd.

agree with you.